docs/source/azure/advanced/azurefunctions.rst

```diff
@@ -34,7 +44,26 @@ provisioned. Application Insights has six available locations and thus can not a
 region as the other resources: West US 2, East US, North Europe, South Central US, Southeast Asia, and
 West Europe. The sku, skuCode, and workerSize correlate to scaling up the App Service Plan.
 
-An example on how to set the provision-options when running in azure functions mode:
+An example on how to set the servicePlanName and accept defaults for the other values:
+
+.. code-block:: yaml
+
+    policies:
+      - name: stopped-vm
+        mode:
+            type: azure-periodic
```

Comment:
I'm not super familiar with azure functions so let me know if this isn't applicable, but user-definable schedules are available in the original `periodic` mode, is there a way for the user to define the cron schedule or rate of execution wrt azure periodic functions?

example
```yaml
policies:
  - name: periodic-ec2
    resource: ec2
    mode:
       type: periodic
       schedule: rate(5 minutes)
...
```

Reply:
Yeah, it is just missing in this example :)   Let me update it with the field documented!  Tx!